driverconfig/config.yaml

```diff
@@ -112,6 +113,13 @@ drivers:
         external-health-monitor: gcr.io/k8s-staging-sig-storage/csi-external-health-monitor-controller:v0.5.0
   - name: isilon
     configVersions:
+      - configVersion: v2.6.0
+        useDefaults: true
+        supportedVersions:
+          - version: v122
+          - version: v124
+          - version: v125
+          - version: v126
       - configVersion: v2.5.0
         useDefaults: true
```

Comment:
Set this flag to false and set the sidecar versions accordingly

Reply:
@rensyct - Modified the Flag to False and added the Side car to the Previous version.